src/backend/SyncBackendConnection.cpp

```diff
@@ -1149,4 +1149,11 @@ std::string SyncBackendConnection::get_data_kind_units(
     }
 }
 
+std::vector<std::string> SyncBackendConnection::ds_supported_transports()
+{
+    std::vector<std::string> supported_transports = {"UDPv4", "UDPv6", "TCPv4", "TCPv6"};
+
+    return supported_transports;
```

Comment:
```suggestion
    return std::vector<std::string>({"UDPv4", "UDPv6", "TCPv4", "TCPv6"});
```
Even when the compiler should take care of this, it is better to make it explicit,